scripts/Moon.gd

```diff
@@ -1,4 +1,24 @@
 extends Node2D
 
+class_name Moon
+
+var face = 1.0
+
+var hunger_duration_seconds = 30.0
+
+func _ready():
+	set_crescent(1.0)
+
+func _process(delta):
+	amend_crescent((1.0 / hunger_duration_seconds) * -delta)
```

Comment:
We need negative delta handling :stuck_out_tongue:

Reply:
b8226a4